apps/spruce/cypress/integration/version/version_timing.ts

```diff
@@ -0,0 +1,174 @@
+import { clickOnPageSizeBtnAndAssertURLandTableSize } from "../../utils";
+
+describe("Version Timing Tab without a variant selected", () => {
+  beforeEach(() => {
+    cy.visit("/version/5e4ff3abe3c3317e352062e4/version-timing");
+  });
+  it("shows a chart of all variants in the version", () => {
```

Comment:
This doesn't show all the variants. The only visible variant in this test is Ubuntu 16.04. I found that out by running the test with `cy.pause()` at the bottom of the test body.